scripts/background.js

```diff
@@ -1150,7 +1140,7 @@ function openThatContext(temp,url,methodOfShowing){
   var whois_url="https://archive.org/services/context/whois?url=" + url;
   var tweet_url="https://archive.org/services/context/twitter?url="+url;
   var hoaxy_url="http://hoaxy.iuni.iu.edu/#query="+url+"&sort=mixed&type=Twitter";
-  if(methodOfShowing=='tab'){
+  if(methodOfShowing=='tab'){ 
```

Comment:
Watch out for unnecessary trailing spaces, vim has many ways to highlight them in order to avoid having them,
there are ways to do it in all major editors.